src/contracts/pods/EigenPodManager.sol

```diff
@@ -177,6 +176,19 @@ contract EigenPodManager is Initializable, OwnableUpgradeable, Pausable, IEigenP
         _setMaxPods(newMaxPods);
     }
 
+    function decrementWithdrawableRestakedExecutionLayerGwei(address podOwner, uint256 amountWei) 
+        external onlyStrategyManager onlyWhenNotPaused(PAUSED_WITHDRAW_RESTAKED_ETH) 
+    {
+        ownerToPod[podOwner].decrementWithdrawableRestakedExecutionLayerGwei(amountWei);
+    }
+
+    function incrementWithdrawableRestakedExecutionLayerGwei(address podOwner, uint256 amountWei) 
+        external onlyStrategyManager onlyWhenNotPaused(PAUSED_WITHDRAW_RESTAKED_ETH) 
```

Comment:
is this the correct 'pause' modifier / is it being applied in the appropriate place in the call stack?

Reply:
yup, I believe it is.  Should have similar behaviour to eigenPodManager.withdrawRestakedBeaconChainETH I would think

Reply:
OK, agreed this seems right